frontend/src/console/components/RedeploymentToolbar/RedeploymentToolbar.tsx

```diff
@@ -25,6 +25,10 @@ export const RedeploymentToolbar: React.FC<Props> = ({
 }: Props) => {
   const [show, setShow] = React.useState(false);
 
+  const [alertShow, setAlertShow] = React.useState(true);
+
+  const handleCloseAlert = () => setAlertShow(false);
```

Comment:
I think this functionality is meaningless since it's not persisted on page refresh. Let it be a persistent notification till the email is verified, since we want to gate access on this basis.

Reply:
Thanks, I have updated it :)